src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Create.cs

```diff
@@ -123,69 +113,6 @@ internal sealed partial class SyntaxTreeIndex
                             containsImplicitObjectCreation = containsImplicitObjectCreation || syntaxFacts.IsImplicitObjectCreationExpression(node);
                             containsGlobalAttributes = containsGlobalAttributes || syntaxFacts.IsGlobalAttribute(node);
                             containsConversion = containsConversion || syntaxFacts.IsConversionExpression(node);
-
-                            if (syntaxFacts.IsUsingAliasDirective(node) && infoFactory.TryGetAliasesFromUsingDirective(node, out var aliases))
```

Comment:
aliases were used only inside the 'DeclaredSymbolInfo' computation.  so all this moved into the other type.